### DISCUSSION TITLE
Can't seem to open it on my iMac Pro after installation.

### DISCUSSION THREAD

**Discussion by iannwu** [smol-ai/GodMode#167] [Q&A]
Any suggestions on how I can get it to work? Thanks!

<img width="261" alt="Screenshot 2023-08-21 at 10 14 12 AM" src="https://github.com/smol-ai/GodMode/assets/5016913/388975d1-e796-4dcd-8e4e-6f380f309947">
<img width="279" alt="Screenshot 2023-08-21 at 10 14 25 AM" src="https://github.com/smol-ai/GodMode/assets/5016913/6f500854-ea4f-477b-94f4-320cddd7fb43">

**Comment by kontinue2learn** [comment]
Same problem here.

**Reply by TimeScripter** [reply]
same here. I have reinstalled and installed with the latest version of mac. is there any other settings that should be tweaked?
